lib/glific/notion.ex

```diff
@@ -0,0 +1,292 @@
+defmodule Glific.Notion do
+  @moduledoc """
+  Notion API integration Utilities
+  """
+  alias Glific.Registrations.Registration
+
+  @doc """
+  Utilities to interact with Notion
+  """
+
+  @ngo_database_id "880cf3440d4c4e3f80453103e712f897"
+
+  @notion_base_url "https://api.notion.com/v1"
+  require Logger
+  use Tesla
+
+  plug(Tesla.Middleware.JSON, engine_opts: [keys: :atoms])
+
+  @spec headers :: list()
+  defp headers,
+    do: [
+      {"Content-Type", "application/json"},
+      {"Authorization", "Bearer " <> Application.get_env(:glific, :notion_secret)},
+      {"Notion-Version", "2022-02-22"}
+    ]
+
+  @doc """
+  Creates a new entry in notion ngo onboarded database
+  """
+  @spec create_database_entry(map()) :: {:ok, String.t()} | {:error, String.t()}
+  def create_database_entry(properties) do
```

Comment:
check if `database` is the terminology used in notion

Reply:
Yes, https://developers.notion.com/docs/working-with-databases